### DISCUSSION TITLE
Can't get my site images to work on particular posts (Jekyll and Github Pages)

### DISCUSSION THREAD

**Discussion by AgentArachnid66** [community/community#124804] [Programming Help]
### Select Topic Area

General

### Body

This is really weird as my images are formatted the same across my site so I know the URLs are good. The problem is arising from my Personal Projects folder not being able to display images, in particular my Inventory and Quest posts. 

[Inventory](https://agentarachnid66.github.io/PersonalProjects/Inventory/)

I've copied over an image URL that is working elsewhere on my page into the Inventory post and it is still not showing up. 
https://github.com/AgentArachnid66/AgentArachnid66.github.io

I'm using Jekyll for the site generation

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @AgentArachnid66 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by YuheshPandian** [comment]
But i can view images on the following two site you mentioned...
